lua/dooing/ui/windows/main.lua

```diff
@@ -0,0 +1,194 @@
+local M = {}
+
+-- Dependencies
+local config = require("dooing.config")
+local state = require("dooing.state")
+
+-- Local window IDs and buffers
+local win_id = nil
+local buf_id = nil
+
+-- Creates and configures the small keys window
+local function create_small_keys_window(main_win_pos)
+	if not config.options.quick_keys then
+		return nil
+	end
+
+	local small_buf = vim.api.nvim_create_buf(false, true)
+	local width = config.options.window.width
+
+	-- Define two separate line arrays for each column
+	local lines_1 = {
+		"",
+		"  i - New todo",
+		"  x - Toggle todo",
+		"  d - Delete todo",
+		"  u - Undo delete",
+		"  H - Add due date",
+		"",
+	}
+
+	local lines_2 = {
+		"",
+		"  T - Add time",
+		"  t - Tags",
+		"  / - Search",
+		"  I - Import",
+		"  E - Export",
+		"",
+	}
+
+	-- Calculate middle point for even spacing
+	local mid_point = math.floor(width / 2)
+	local padding = 2
+
+	-- Create combined lines with centered columns
+	local lines = {}
+	for i = 1, #lines_1 do
+		local line1 = lines_1[i] .. string.rep(" ", mid_point - #lines_1[i] - padding)
+		local line2 = lines_2[i] or ""
+		lines[i] = line1 .. line2
+	end
+
+	vim.api.nvim_buf_set_lines(small_buf, 0, -1, false, lines)
+	vim.api.nvim_buf_set_option(small_buf, "modifiable", false)
+	vim.api.nvim_buf_set_option(small_buf, "buftype", "nofile")
+
+	-- Position it under the main window
+	local row = main_win_pos.row + main_win_pos.height + 1
+
+	local small_win = vim.api.nvim_open_win(small_buf, false, {
+		relative = "editor",
+		row = row,
+		col = main_win_pos.col,
+		width = width,
+		height = #lines,
+		style = "minimal",
+		border = "rounded",
```

Comment:
The config defines settings such as border style. To ensure consistent theming between windows, all windows should read such values from the config itself (at least for the style and borders, height and width are window-dependent)